gtsfm/multi_view_optimizer.py

```diff
@@ -42,6 +46,10 @@ def __init__(
         self.ba_optimizer = bundle_adjustment_module
         self._run_view_graph_estimator: bool = self.view_graph_estimator is not None
```

Comment:
We should use the flag for stage 2 too?

Reply:
@ayushbaid did you mean to add the MEDIAN_EDGE_ERROR stage to the config also?

Reply:
I meant to make it optional. If we do not want to run stage 2 we can easily skip it. 

But if we get good performance across the board and there is no reason to not use it, I am fine.